examples/trials/benchmarking/automlbenchmark/setup.sh

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
I think `#!/bin/bash` won't work if you add extra comment lines here.

Reply:
It works in my local machine though. I will remove the comments.